tests/git-auto-commit.bats

```diff
@@ -43,26 +57,27 @@ teardown() {
     rm -rf "${FAKE_LOCAL_REPOSITORY}"
     rm -rf "${FAKE_REMOTE}"
     rm -rf "${FAKE_TEMP_LOCAL_REPOSITORY}"
+    rm "${GITHUB_OUTPUT}"
 }
 
 # Create a fake remote repository which tests can push against
 _setup_fake_remote_repository() {
     # Create the bare repository, which will act as our remote/origin
-    rm -rf "${FAKE_REMOTE}";
```

Comment:
I can revert these changes if you care. VSCode + shell-format extension naively removed a bunch of these.

Reply:
That's fine. I just noticed that the bats file wasn't very consistent. Better to ditch all those `;`.